### PR TITLE
Make constants out of pxe annotations

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

File: src/constants/index.js [hunk 1]
```diff
@@ -31,6 +31,9 @@ export const PROVISION_SOURCE_TEMPLATE = 'Template';
 
 export const POD_NETWORK = 'Pod Networking';
 
+export const ANNOTATION_PXE_INTERFACE = 'cnv.ui.pxeInterface';
```
Comment (mareklibra) [inline src/constants/index.js]: nitpick: Is this constant explicitly defined somewhere in documentation (or so)?
Does it make sense to add reference link as a comment here?

Same as the constants above.

Comment (mareklibra) [inline src/constants/index.js]: nitpick: can we order the lines alphabetically or at least group them by context (like `TEMPLATE_` or `ANNOTATION_`)?

Comment (rawagner) [reply src/constants/index.js]: no, I just made it up. This is used only by us (UI) for later VM edit

Comment (rawagner) [reply src/constants/index.js]: will do!

**Comment by ohadlevy** [comment]
:+1: does it make sense to extract other things such as virtio etc?

File: src/constants/index.js (hunk 1 above)
Comment (rawagner) [reply src/constants/index.js]: For other constants..some of them are used in common-templates, but there are no docs, we could link to the common-templates repo ..?
Others are used in kubevirt itself (like API_VERSION)..we could link to user guide's VM example.. but that does not seem correct..

**Comment by rawagner** [comment]
@ohadlevy virtio is already a constant. I added a few more from strings that are used in `k8s/request.js`. If you have any suggestions let me know.

File: src/constants/index.js (hunk 1 above)
Comment (rawagner) [reply src/constants/index.js]: done

**Pull request merged**